README.md

```diff
@@ -2,11 +2,82 @@
 
 The `zcashd_exporter` will poll a zcashd node's rpc endpoint, retreive data, then preent it for prometheus to scrape.
 
-## Getting started
+## Getting started locally
 
 ```
 go get -v github.com/zcash-hackworks/zcashd_exporter
 go install github.com/zcash-hackworks/zcashd_exporter
 $GOPATH/bin/zcashd_exporter --help
 ```
 
+## Getting Started With docker-compose
+
+#### Create zcashd direcotries
```

Comment:
I was thrown a bit by the jump from H2 to H4 headers. Should this one (and the following one) be H3?

Reply:
Ah, fixed the headings